src/appl/jj2lsj90/jj2lsj_code.f90

```diff
@@ -2897,7 +2900,7 @@ SUBROUTINE uniquelsj
             END DO
           end if
           IPRGG(Lev_OPT) = IPR + IPRGG(Lev_OPT) - 1
-          if(IPRGG(Lev_OPT) >= MAS_MAX(Lev_OPT)                        &
+          if(IPRGG(Lev_OPT) > MAS_MAX(Lev_OPT)                        &
```

Comment:
Having `>=` here is a bug and is incorrect?

Reply:
I suppose so, again, I trust Gediminas here. We'll discuss the details during a video chat down the line.